lib/beamchmark/suite/memory/memory_task.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Beamchmark.Suite.Memory.MemoryTask do
+  @moduledoc """
+  This module contains the memory benchmarking task.
+  Measurements are performed using [`:erlang.memory/0`](https://www.erlang.org/doc/man/erlang.html#memory-0)
+  Run example:
+  ```
+  MemoryTask.start_link()
+  ```
```

Comment:
I don't really get this "run example", both here and in `CpuTask`:

- `start_link` takes 2 arguments, not 0.
- They explain very basic thing, meanwhile we have more sophisticated examples in the `examples` folder.

Reply:
I removed those examples.